.github/workflows/ci.yml

```diff
@@ -44,3 +49,6 @@ jobs:
         run: npm cit
         env:
             CI: true
+
+      - name: Perform CodeQL Analysis
```

Comment:
The `eslint` block and the `test` block actually run on different machines, so this call to perform the CodeQL Analysis should go right under the lines to Initialize CodeQL :)